mod-source-record-storage-server/src/main/java/org/folio/services/handlers/AbstractPostProcessingEventHandler.java

```diff
@@ -72,14 +74,19 @@ public abstract class AbstractPostProcessingEventHandler implements EventHandler
   private static final String FAILED_UPDATE_STATE_MSG = "Error during update records state to OLD";
   private static final String ID_FIELD = "id";
   public static final String POST_PROCESSING_INDICATOR = "POST_PROCESSING";
+  public static final String CENTRAL_TENANT_INSTANCE_UPDATED_FLAG = "CENTRAL_TENANT_INSTANCE_UPDATED";
```

Comment:
In general, I don't like these special values straddling mod-inventory and SRS. Given how DI works today, it is necessary to propagate the appropriate data in the context between the two modules.

Reply:
@okolawole-ebsco  So we decided to provide these "indicators" in the payload context and they will be removed after using. (See saveRecordForCentralTenant() method). 
I don't understand you correctly. 
Could you please describe - where should we provide these values? If not inside payloadContext? Thank you!

Reply:
I dont have any other suggestions. I understand why it was done. When we refactor some stuff in the coming release, we might come up with something better. No actions neccessary.

Reply:
Where I am thinking is that we would put something like this in data-import-processing-core.

Reply:
@okolawole-ebsco  Oh, I see. I remember it.